docs/admin/install/eventing/install-eventing-with-yaml.md

```diff
@@ -90,6 +90,60 @@ Follow the procedure for the Channel of your choice:
 
         <!-- TODO(https://github.com/knative/docs/issues/2153): Add more Channels here -->
 
+### Setting a default Channel
+
+If you are using different channel implementations, such as KafkaChannel, or you want a specific configuration of the InMemoryChannel to be the default configuration, you can change the default behavior by updating the `default-ch-webhook` ConfigMap.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: default-ch-webhook
+  namespace: knative-eventing
+data:
+  default-ch-config: |
+  clusterDefault:
+      apiVersion: messaging.knative.dev/v1beta1
+      kind: KafkaChannel
+      spec:
+      numPartitions: 10
+      replicationFactor: 1
+  namespaceDefaults:
+      my-namespace:
+      apiVersion: messaging.knative.dev/v1
+      kind: InMemoryChannel
+      spec:
+          delivery:
+          backoffDelay: PT0.5S
+          backoffPolicy: exponential
+          retry: 5
+```
```

Comment:
I'd add some comments or call outs here that explain what's being updated and what it does?